components/Players/PlayerManager.vue

```diff
@@ -564,6 +570,13 @@ export default Vue.extend({
           ]
         });
       }
+    },
+    toggleFullScreen(): void {
+      this.$fullscreen.toggle(this.$refs.playerContainer as Element, {
```

Comment:
@Camc314 We didn't need $el here because the element we're using as a ref here is a void HTML element, not a Vue component.

Reply:
Without $el, this broke for me. 

The `toggle` function expects the HTML element, `this.$refs.playerContainer` is of type vue element. $el is the HTML element